Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage.py

```diff
@@ -310,53 +401,116 @@ def get_attachment_command() -> None:
-    else:
-        return response
-
-
 def get_report_by_id_command() -> None:
-    # arguments importing
     report_id = int(demisto.getArg('report_id'))  # type: int
     verbose = demisto.getArg('verbose') == "true"
 
-    # running the command
-    res = get_report_by_id(report_id)[0]
+    report = TriageReport.fetch(report_id)
 
-    # parsing outputs
-    if not verbose:
-        # extract only relevant fields
-        res = {k: res[k] for k in res.keys() & TERSE_FIELDS}
-
-    # get the report body, and create html file if necessary
-    if res:
-        parse_report_body(res)
-        res['reporter'] = get_reporter(res.get('reporter_id'))  # enrich: id -> email
-        hr = tableToMarkdown("Report Summary:", res, headerTransform=split_snake, removeNull=True)
-        ec = {'Cofense.Report(val.ID && val.ID == obj.ID)': snake_to_camel_keys([res])}
-        return_outputs(readable_output=hr, outputs=ec)
+    if not report:
+        return return_error('Could not find report with matching ID')
 
+    if verbose:
+        report_attrs = report.attrs
     else:
-        return_error('Could not find report with matching ID')
+        report_attrs = report.terse_attrs
 
-
-def parse_report_body(report) -> None:
-    if 'report_body' in report and 'HTML' in report['report_body']:
-        attachment = fileResult(
-            filename=f'{report.get("id")}-report.html',
-            data=report.get('report_body').encode(),
+    if report.attachment:
+        demisto.results(
+            {
+                **report.attachment,
+                **{"HumanReadable": "### Cofense HTML Report:\nHTML report download request has been completed"},
+            }
         )
-        attachment['HumanReadable'] = '### Cofense HTML Report:\nHTML report download request has been completed'
-        demisto.results(attachment)
-        del report['report_body']
+        del report_attrs["report_body"]
+
+    hr = tableToMarkdown("Report Summary:", report_attrs, headerTransform=split_snake, removeNull=True)
+    ec = {'Cofense.Report(val.ID && val.ID == obj.ID)': snake_to_camel_keys([report_attrs])}
+    return_outputs(readable_output=hr, outputs=ec)
+
+
+def get_threat_indicators_command() -> None:
+    results = TRIAGE_INSTANCE.request(
+        "triage_threat_indicators",
+        params={
+            "type": demisto.getArg("type"),
+            "level": demisto.getArg("level"),
+            "start_date": demisto.getArg("start_date"),
+            "end_date": demisto.getArg("end_date"),
+            "page": demisto.getArg("page"),
+            "per_page": demisto.getArg("per_page"),
+        },
+    )
+
+    if not results:
+        return return_outputs("no results were found.", {})
+
+    demisto.results(
+        {
+            "Type": entryTypes["note"],
+            "ContentsFormat": formats["markdown"],
+            "Contents": results if results else "no results were found",
```

Comment:
you handled the no results case a few lines up the code
```suggestion
            "Contents": results,
```